integration-tests/keycloak-authorization/src/main/resources/META-INF/native-image/io.quarkus/quarkus-integration-test-keycloak-authorization/native-image.properties

```diff
@@ -0,0 +1,3 @@
+# We need to initialize SecretGenerator at runtime because SecretGenerator has a static field "SecureRandom SECURE_RANDOM = new SecureRandom()"
+# which cannot be initialized at build time during native build.
+Args=--initialize-at-run-time=org.keycloak.common.util.SecretGenerator
```

Comment:
@Eng-Fouad I believe it has to be done in https://github.com/quarkusio/quarkus/blob/main/extensions/keycloak-authorization/deployment/src/main/java/io/quarkus/keycloak/pep/deployment/KeycloakReflectionBuildStep.java

Reply:
@sberyozkin OK, I will move it there.